etl/steps/data/garden/news/2024-05-08/guardian_mentions.py

```diff
@@ -25,35 +46,86 @@ def run(dest_dir: str) -> None:
         df=tb,
         countries_file=paths.country_mapping_path,
     )
-    tb = geo.harmonize_countries(
-        df=tb,
-        countries_file=paths.country_mapping_path,
-    )
 
     ## Get relative values
-    tb = add_relative_indicator(tb, "num_pages_tags")
-    tb = add_relative_indicator(tb, "num_pages_mentions")
+    tb = add_relative_indicators(tb, ["num_pages_tags", "num_pages_mentions"])
 
-    tb = tb.format(["country", "year"])
+    ## Add per-capita indicators
+    tb = geo.add_population_to_table(tb, ds_population)
+    for column in ["num_pages_tags", "num_pages_mentions"]:
+        tb[f"{column}_per_million"] = tb[column] / tb["population"] * 1_000_000
+    tb = tb.drop(columns="population")
 
+    # Estimate 10-year average
+    tb_10y_avg = tb[(tb["year"] >= 2014) & (tb["year"] <= 2023)].copy()
+    tb_10y_avg = tb_10y_avg.rename(
+        columns={col: f"{col}_10y_avg" for col in tb_10y_avg.columns if col not in COLUMN_INDEX}
+    )
+    columns_indicators = [col for col in tb_10y_avg.columns if col not in COLUMN_INDEX]
+    tb_10y_avg = tb_10y_avg.groupby("country")[columns_indicators].mean().reset_index()
+    tb_10y_avg["year"] = 2023
+
+    # Estimate log(10-year average)
+    tb_10y_avg_log = tb_10y_avg.copy()
+    tb_10y_avg_log[columns_indicators] = np.log(tb_10y_avg[columns_indicators] + 1)
+    tb_10y_avg_log = tb_10y_avg_log.rename(columns={col: f"{col}_10y_avg_log" for col in columns_indicators})
+    tb_10y_avg_log["year"] = 2023
+
+    ## Format
+    tb = tb.format(COLUMN_INDEX)
+    tb_10y_avg = tb_10y_avg.format(["country", "year"], short_name="avg_10y")
+    tb_10y_avg_log = tb_10y_avg_log.format(["country", "year"], short_name="avg_log_10y")
+
+    tables = [
+        tb,
+        tb_10y_avg,
+    ]
     #
     # Save outputs.
     #
     # Create a new garden dataset with the same metadata as the meadow dataset.
     ds_garden = create_dataset(
-        dest_dir, tables=[tb], check_variables_metadata=True, default_metadata=ds_meadow.metadata
+        dest_dir, tables=tables, check_variables_metadata=True, default_metadata=ds_meadow.metadata
     )
 
     # Save changes in the new garden dataset.
     ds_garden.save()
 
 
+def add_relative_indicators(tb, colnames):
```

Comment:
Remember to set 'types' for the function arguments e.g. `tb:Table` - that's still best practice right?